packages/framework/src/Resources/scripts/admin/customerForm.js

```diff
@@ -1,8 +1,8 @@
 (function ($) {
 
     Shopsys.register.registerCallback(function ($container) {
-        var $select = $container.filterAllNodes('#customer_form_userData_pricingGroup');
-        var $control = $container.filterAllNodes('#customer_form_userData_domainId');
+        var $select = $container.filterAllNodes('.js-toggle-opt-group-select');
+        var $control = $container.filterAllNodes('.js-toggle-opt-group-control');
 
         if ($select.length > 0 && $control.length > 0) {
```

Comment:
This looks like it *should* work for multiple elements on one page but doesn't. Better solution would be to have `js-toggle-opt-group` class on the `ChoiceType` for pricing group with a data attribute `data-toggle-opt-group-control-selector`. Now, it would be linked together, `.js-toggle-opt-group` could be iterated by a `$.each()` and you could either call `Shopsys.selectToggle.toggleOptgroupOnControlChange($(this), $($(this).data('toggle-opt-group-control-selector')))` or refactor it to work with the data attribute itself.

Also, it should be a part of `selectToggle.js` now because it is generalized - now it has no direct link to customer form.

Reply:
Ok, i created it into new commit cf3ad81e1bcaeda484dec48a6d85bf09d173104b